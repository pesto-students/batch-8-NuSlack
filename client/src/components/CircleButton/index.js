import React from 'react';
import { Button } from 'antd';

const index = props => {
  return <Button type="circle" size="default" {...props} />;
};

export default index;
