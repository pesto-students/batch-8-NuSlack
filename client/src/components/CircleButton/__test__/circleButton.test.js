import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import CircleButton from '../index';

describe('<CircleButton />', () => {
  it('should match the snapshot', () => {
    const tree = shallow(<CircleButton size="default" />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
