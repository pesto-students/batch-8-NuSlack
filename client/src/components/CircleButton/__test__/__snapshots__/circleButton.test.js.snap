// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CircleButton /> should match the snapshot 1`] = `
<Button
  block={false}
  ghost={false}
  htmlType="button"
  loading={false}
  size="default"
  type="circle"
/>
`;
